modules/planning/tasks/optimizers/piecewise_jerk_path/piecewise_jerk_path_optimizer.cc

```diff
@@ -170,15 +170,15 @@ bool PiecewiseJerkPathOptimizer::OptimizePath(
                                         FLAGS_lateral_derivative_bound_default);
   piecewise_jerk_problem.set_dddx_bound(FLAGS_lateral_jerk_bound);
 
-  /**
-  // Experimental code to be tested
-  // TODO(all): find the params in vehicle config
-  double axis_distance = 2.5;
-  double max_steering_rate = 1.0 / 6.0 * M_PI;
+  // Estimate jerk boundary from vehicle_params
+  const auto& veh_param =
+      common::VehicleConfigHelper::GetConfig().vehicle_param();
+  double axis_distance = veh_param.wheel_base();
+  double max_steering_rate = veh_param.max_steer_angle_rate() /
```

Comment:
Did you mean heading angle change rate here?

Reply:
Or wheel angle change rate?

Reply:
fixed, thanks!